Formula/scm-manager.rb

```diff
@@ -5,6 +5,11 @@ class ScmManager < Formula
   sha256 "8861aa4c765dfe046264f97347065c9c1c6055c14abf4b8d09a340f393a41b29"
   license all_of: ["Apache-2.0", "MIT"]
 
+  livecheck do
+    url "https://scm-manager.org/download/rss.xml"
+    regex(%r{/download/(\d+(?:\.\d+)+)}i)
```

Comment:
```suggestion
    url "https://scm-manager.org/download/"
    regex(/href=.*?unix[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

I would go with the download page in this case, as it allows us to match the version from the tarball that we use in the formula's `stable` URL. The RSS feed seems to only announce the release of a version, so there's no guarantee that the tarball we need will be available at the same time the release post is created. It's likely fine but there's something to be said for matching the version from the tarball filename when we have options.

Reply:
sometimes, it is the other way around, the tarball is in the place, but the release did not get announced. I guess we will find it out later for this formula.